docs/src/main/sphinx/functions/list-by-topic.rst

```diff
@@ -0,0 +1,548 @@
+* :func:`ST_Within`
+* :func:`ST_X`
+* :func:`ST_XMax`
+* :func:`ST_XMin`
+* :func:`ST_Y`
+* :func:`ST_YMax`
+* :func:`ST_YMin`
+* :func:`to_encoded_polyline`
+* :func:`to_geojson_geometry`
+* :func:`to_geometry`
+* :func:`to_spherical_geography`
+
+HyperLogLog
+-----------
+
+For more details, see :doc:`hyperloglog`
+
+* :func:`approx_set`
+* ``cardinality()``
+* :func:`empty_approx_set`
+* :func:`merge`
+
+JSON
+----
+
+For more details, see :doc:`json`
+
+* :func:`is_json_scalar`
+* :func:`json_array_contains`
+* :func:`json_array_length`
+* :func:`json_extract`
+* :func:`json_format`
+* :func:`json_parse`
+* :func:`json_size`
+
+Lambda
+------
+
+For more details, see :doc:`lambda`
+
+* :func:`any_match`
+* :func:`reduce_agg`
+* :func:`regexp_replace`
+* :func:`transform`
+
+Machine learning
+----------------
+
+For more details, see :doc:`ml`
+
+* :func:`classify`
+* :func:`features`
+* :func:`learn_classifier`
+* :func:`learn_libsvm_classifier`
+* :func:`learn_libsvm_regressor`
+* :func:`learn_regressor`
+* :func:`regress`
+
+Map
+---
+
+For more details, see :doc:`map`
+
+* :func:`cardinality`
+* :func:`element_at`
+* :func:`map`
+* :func:`map_concat`
+* :func:`map_entries`
+* :func:`map_filter`
+* :func:`map_from_entries`
+* :func:`map_keys`
+* :func:`map_values`
+* :func:`map_zip_with`
+* :func:`multimap_from_entries`
+* :func:`transform_keys`
+* :func:`transform_values`
+
+Math
+----
+
+For more details, see :doc:`math`
+
+* :func:`abs`
+* :func:`acos`
+* :func:`asin`
+* :func:`atan`
+* :func:`beta_cdf`
+* :func:`cbrt`
+* :func:`ceil`
+* :func:`cos`
+* :func:`cosh`
+* :func:`cosine_similarity`
+* :func:`degrees`
+* :func:`e`
+* :func:`exp`
+* :func:`floor`
+* :func:`from_base`
+* :func:`infinity`
+* :func:`inverse_beta_cdf`
+* :func:`inverse_normal_cdf`
+* :func:`is_finite`
+* :func:`is_nan`
+* :func:`ln`
+* :func:`log`
+* :func:`log2`
+* :func:`log10`
+* :func:`mod`
+* :func:`nan`
+* :func:`normal_cdf`
+* :func:`pi`
+* :func:`pow`
+* :func:`power`
+* :func:`radians`
+* :func:`rand`
+* :func:`random`
+* :func:`round`
+* :func:`sign`
+* :func:`sin`
+* :func:`sqrt`
+* :func:`tan`
+* :func:`tanh`
+* :func:`to_base`
+* :func:`truncate`
+* :func:`width_bucket`
+* :func:`wilson_interval_lower`
+* :func:`wilson_interval_upper`
+
+Quantile digest
+---------------
+
+For more details, see :doc:`qdigest`
+
+* ``merge()``
+* :func:`qdigest_agg`
+* :func:`value_at_quantile`
+* :func:`values_at_quantiles`
+
+Regular expression
+------------------
+
+For more details, see :doc:`regexp`
+
+* :func:`regexp_count`
+* :func:`regexp_extract`
+* :func:`regexp_extract_all`
+* :func:`regexp_like`
+* :func:`regexp_position`
+* :func:`regexp_replace`
+* :func:`regexp_split`
+
+Session
+-------
+
+For more details, see :doc:`session`
+
+* :data:`current_catalog`
+* :data:`current_schema`
+* :data:`current_user`
+
+String
+------
+
+For more details, see :doc:`string`
+
+* :func:`chr`
+* :func:`codepoint`
+* :func:`concat`
+* :func:`concat_ws`
+* ``format()``
```

Comment:
This is under conversion, though it's fine to have here also, but make it a link
```
* :func:`format`